transformer_engine/pytorch/ops/basic/activation.py

```diff
@@ -389,3 +390,36 @@ def _activation_forward_impl(self, *args, **kwargs) -> torch.Tensor:
 
     def _activation_backward_impl(self, *args, **kwargs) -> torch.Tensor:
         return tex.dswiglu(*args, **kwargs)
+
+
+class GptOssSwiglu(_ActivationOperation):
```

Comment:
Nit: [The original SwiGLU paper](https://arxiv.org/pdf/2002.05202) calls it "SwiGLU".
```suggestion
class GptOssSwiGLU(_ActivationOperation):
```
Stylistically I don't like that we've camel-cased "GPT" and "OSS", but I guess "GPTOSSSwiGLU" would be overwhelming.